packages/patternfly-4/react-core/src/components/EmptyState/EmptyState.tsx

```diff
@@ -3,12 +3,14 @@ import { css, getModifier } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/EmptyState/empty-state';
 
 export enum EmptyStateVariant {
+  extraLarge = 'extraLarge',
   large = 'large',
   small = 'small',
   full = 'full'
 }
 
 const maxWidthModifiers: { [variant in keyof typeof EmptyStateVariant]: string } = {
+  extraLarge: 'xl',
```

Comment:
I believe we use xLarge in the grid component. We should follow that convention

Reply:
It looks like Grid uses the same sm, md, lg, xl convention other components are using.

Reply:
I'm referring to the `extraLarge` prop name. Grid uses prop names like lg, xl, etc.